src/engine/common/math.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 #include <algorithm>
+#include <math.h>
```

Comment:
Maybe `<cmath>` would be better here?

Reply:
It definitely works, however I do get warnings about `narrowing conversion of ‘sin(((double)((float)angle)))’ from ‘double’ to ‘float’`. It does still solve the compilation error though so works for me :)

Reply:
In that case let's go for it! I will take care of the warnings :)
Thanks again for the fix!